deps/rabbit/src/rabbit_definitions.erl

```diff
@@ -587,8 +612,8 @@ do_concurrent_for_all(List, WorkPoolFun) ->
            fun() ->
                    _ = try
                        WorkPoolFun(M)
-                   catch {error, E} -> gatherer:in(Gatherer, {error, E});
-                         _:E        -> gatherer:in(Gatherer, {error, E})
+                   catch {error, E}      -> gatherer:in(Gatherer, {error, E});
+                         _:E:_Stacktrace -> gatherer:in(Gatherer, {error, E})
```

Comment:
Returning the stack trace does not make much sense since the function that uses `gatherer` to perform a work pool operation cannot do much about it, other than log.

So logging at debug level sounds best.